query-service-impl/src/test/java/org/hypertrace/core/query/service/AbstractServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package org.hypertrace.core.query.service;
+
+import com.google.protobuf.GeneratedMessageV3;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.stream.Collectors;
+import org.hypertrace.core.grpcutils.context.RequestContext;
+
+public abstract class AbstractServiceTest<TQueryServiceRequestType extends GeneratedMessageV3> {
+
+  private static final String QUERY_SERVICE_TEST_REQUESTS_DIR = "test-requests";
+
+  protected TQueryServiceRequestType readQueryServiceRequest(String fileName) {
+    String resourceFileName =
+        createResourceFileName(
+            QUERY_SERVICE_TEST_REQUESTS_DIR,
+            fileName); // QUERY_SERVICE_TEST_REQUESTS_DIR + FILE_SEPARATOR + getTestSuiteName() +
```

Comment:
nit: shall we remove this comment? The function `createResourceFileName` internally explain this.

Reply:
done